docs/integrations/sources/amazon-seller-partner.md

```diff
@@ -241,6 +241,21 @@ For now the waiting logic only work for the following streams:
 
 Create a separate connection for streams which usually fail with error above "Failed to retrieve the report..." and disable sync of these streams in the first connection with streams which don't fail because of the error. Adjust the sync time of these two connection to do not overlap. It's recommended to have a time break between syncs in the connections.
 
+### Rate Limit issue for Report Streams
+
+Syncing Report Streams sometimes may fail due to rate limits.
+
+The Amazon Seller Partner source connector makes requests according to the limits mentioned in the [SP API docs](https://developer-docs.amazon.com/sp-api/lang-pt_BR/docs/reports-api-rate-limits).
+But actual Rate Limits could be differ from ones mentioned in the docs. See [Usage Plans and Rate Limits](https://developer-docs.amazon.com/sp-api/docs/usage-plans-and-rate-limits) for more information.
+Depending on actual rate limits the Amazon Seller Partner source connector can receive "rate limited" responses, unfortunately there is no way to find actual rate limits values for account.
+
+We recommend next steps to overcome the rate limits issue:
+
+1. Depending on your amount of data per [Period In Days](https://docs.airbyte.com/integrations/sources/amazon-seller-partner#reference) adjust this value to reduce time of processing the report on API Side. If creation of the report takes more than 1h it's recommended to set lower value for `Period In Days` setting.
```

Comment:
very small nit, but since this is documentation, we should spell our "1 hour" just to keep things consistently as words w/o the abbreviation.

And same for below 24h unless that is the actual input of a config setting.